common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java

```diff
@@ -64,6 +64,8 @@ public class GlueSchemaRegistryConfiguration {
     private List<SerializationFeature> jacksonSerializationFeatures;
     private List<DeserializationFeature> jacksonDeserializationFeatures;
 
+    private boolean logicalTypesConversionEnabled;
```

Comment:
Can it be `final`? 🤔 (I don't remember Java, sorry)

Reply:
no because it's set in the constructor :( (final is `val` in Scala)

Reply:
> (final is val in Scala)

This I remember, thanks 😄

Ok sad story then